source/configure/calls-deployment.rst

```diff
@@ -338,10 +342,35 @@ If you wish to host many calls or calls with a large number of participants, tak
   # Allow to allocate more memory as needed for more control messages that need to be sent for each socket connected
   net.core.optmem_max = 16777216
 
+The rtcd service
+----------------
+
+.. include:: ./calls-rtcd-ent-only.rst
+  :start-after: :nosearch:
+
+The Calls plugin has a built-in `Selective Forwarding Unit (SFU) <https://bloggeek.me/webrtcglossary/sfu/>`__ to route audio and screensharing data. But this SFU functionality can be split from the Calls plugin and hosted on its own, e.g., on its own node in a `kubernetes deployment <#kubernetes-deployments>`__, or its own dedicated server `alongside Mattermost instances <#modes-of-operation>`__ in a cloud service or self-hosted.
+
+Reasons to use the ``rtcd`` service
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It helps to understand when and why your organization would want to use ``rtcd``.
+
+First of all, ``rtcd`` is a standalone service, which adds operational complexity. For most small instances of Mattermost there is no need to use ``rtcd``. Instead, use the Calls plugin until performance or reliability becomes an issue.
+
+- **Performance and stability of the main Mattermost server(s).** Letting the Calls plugin run the SFU means that calls traffic is being processed by the server that's also processing the rest of the Mattermost data (Channels, Boards, Playbooks, etc). If the Calls traffic spikes, it can affect the responsiveness of the rest of the Mattermost services. Using an ``rtcd`` service isolates the calls traffic processing to those ``rtcd`` instances.
```

Comment:
K, added. I'm not sure of the wording though, @justinegeffen can you think of a way of making it less formal? (It reads kind of academic at the moment) (line 362)

Reply:
- **Performance of the main Mattermost server(s).** When the Calls plugin runs the SFU, calls traffic is added to the processing load of the server running the rest of your Mattermost services. If Calls traffic spikes, it can negatively affect the responsiveness of these services. Using an ``rtcd`` service isolates the calls traffic processing to those ``rtcd`` instances, and also reduces costs by minimizing CPU usage spikes.